presidio-anonymizer/tests/test_batch_anonymizer_engine.py

```diff
@@ -0,0 +1,71 @@
+from presidio_anonymizer import BatchAnonymizerEngine
+from presidio_anonymizer.entities import (
+  OperatorConfig, RecognizerResult, DictRecognizerResult)
+
+
+texts = ['John', 'Jill', 'Jack']

```

Comment:
Consider putting these in a pytest fixture, or a parameterized test

Reply:
updated to pytest fixture